lib/modules/manager/helm-values/types.ts

```diff
@@ -18,3 +21,17 @@ export interface HelmDockerImageDependencyVersion
 export type HelmDockerImageDependency =
   | HelmDockerImageDependencyTag
   | HelmDockerImageDependencyVersion;
+
+export const ChartDefinition = z
+  .object({
+    apiVersion: z.string().regex(/v([12])/),
+    name: z.string().min(1),
+    version: z.string().regex(
+      // Copied from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+      /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
+    ),
+  })
```

Comment:
remove the regex, we don't need to be so strict here. a wrong version will be catched on bump version.